ga4gh/exceptions.py

```diff
@@ -0,0 +1,83 @@
+"""
+Module containing error objects we return to clients
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+class ParentException(Exception):
+
+    def __init__(self):
+        super(ParentException, self).__init__()
+        self.message = ""
+        self.code = ""
+        self.description = ""
+        self.httpStatus = None
+
+    def toDict(self):
+        return {
+            "message": self.message,
+            "code": self.code,
+            "description": self.description,
+        }
+
+
+class BadRequestException(ParentException):
+
+    def __init__(self):
+        super(BadRequestException, self).__init__()
+        self.httpStatus = 400
+        self.message = "Bad request"
+        self.code = "GAE1"
```

Comment:
Yeah, the schema should probably change this type to a string for more flexibility... unless they also want to give us defined codes for defined errors (but I think they probably view this as outside the schema's purview)